### PR TITLE
Added AVAX support

### PR DESCRIPTION
**Pull request by canokaue**
Correction of PR #131 , now adding Avalanche support instead of Travala.

## Issue number

https://gitcoin.co/issue/ensdomains/ens-app/913/100023629

## Description

Add AVAX (Avalanche X net) support.
## List of features added/changed

Add AVAX on:
src/index.ts
src/tests/index.test.ts
README.md
Already sorted by Cointype.

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.
https://medium.com/avalabs/updates-to-your-avalanche-address-improving-the-user-experience-for-all-89bd671d096d
https://gist.github.com/cgcardona/35ebfe93a26d76add577927564969f8f
https://github.com/ava-labs/avalanche-wallet/blob/3bc65b45ddb30d14e1f15b22030f3ee619c68ade/src/js/wallets/AvaHdWallet.ts#L46-L47

## How Has This Been Tested?

Local test run
TravisCI
## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by Arachnid** [comment]
I have updated the Travis build to check that entries in the formats, tests, and README lists are in the correct order. Please check your PR and update it to maintain correct ordering in these lists.

File: src/__tests__/index.test.ts [hunk 1]
```diff
@@ -375,23 +375,14 @@ const vectors: Array<TestVector> = [
       { text: 'tz3NdTPb3Ax2rVW2Kq9QEdzfYFkRwhrQRPhX', hex: '0002193b2b3f6b8f8e1e6b39b4d442fc2b432f6427a8' },
     ],
   },
-  {
-    name: 'ADA',
```
Comment (makoto) [inline src/__tests__/index.test.ts:379]: Why do you remove ADA?

Comment (canokaue) [reply src/__tests__/index.test.ts]: I must have confused "ADA" with "AVA" when transitioning the commit from Travala to Avalanche, my bad!

**Comment by canokaue** [comment]
@makoto Fixed the accidental ADA removal and also resolved conflicts.

**Comment by makoto** [comment]
Hi @canokaue do you know why the PR didn't have `X-` prefix?

**Pull request merged**